scripts/malloc-init.pl

```diff
@@ -1,70 +0,0 @@
-#!/usr/bin/perl
```

Comment:
This script is trying to identify if we have malloc'ed something but did not initialise it with memset, memcpy,fread, etc. However, this is now a little redundant because we have migrated to calloc which initialises everything to 0 by default.